flow/include/flow/genericactors.actor.h

```diff
@@ -100,6 +100,21 @@ T sorted(T range) {
 	return range;
 }
 
+template <class T>
+std::vector<T> parseStringToVector(std::string str, char delim) {
```

Comment:
This should probably take `str` by const reference and IMO this doesn't belong in genericactors.actor.h